feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts

```diff
@@ -117,9 +97,6 @@ export class ConfiguratorAttributeNumericInputFieldComponent
       .subscribe(() => this.onChange());
   }
 
-  /**
-   * Hit when user input was changed
-   */
   onChange(): void {
```

Comment:
The parent class also has `onChange`, the only difference is this `onChange` has the check `if (!this.attributeInputForm.invalid)`. So, I think we can reuse the parent function (also remove the function `createEventFromInput()`

Reply:
Done!